.github/workflows/test.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Hewlett Packard Enterprise Development LP
+
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  test:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        go: [ '1.24' ]
```

Comment:
Would be good to bump the `go.mod` version to `1.24` too

Reply:
Yup. PR coming up